crates/client/src/client/schema/chain.rs

```diff
@@ -418,6 +416,20 @@ impl TryFrom<GasCosts> for fuel_core_types::fuel_tx::GasCosts {
                     xori: value.xori.into(),
 
                     aloc: value.aloc_dependent_cost.into(),
+                    bsiz: value
+                        .bsiz
+                        .unwrap_or(DependentCost::LightOperation(LightOperation {
```

Comment:
Usage of `LightOperation` with zero `units_per_gas` will cause a panic(division by zero). Please use `DependentCost::free` instead